### PR TITLE
Add Move Up / Move Down buttons 

### PR DESCRIPTION
**Pull request by vutoan1245**
Screen Shot 2020-03-16 at 10 10 55 AM

Resolve issue #1171

### CODE REVIEW

File: packages/app/obojobo-document-engine/src/scripts/oboeditor/components/navigation/more-info-box.js [hunk 1]
```diff
@@ -241,14 +241,8 @@ class MoreInfoBox extends React.Component {
...
-								{this.props.isFirst ? null : (
-									<Button onClick={() => this.props.moveNode(this.props.index - 1)}>Move Up</Button>
-								)}
-								{this.props.isLast ? null : (
-									<Button onClick={() => this.props.moveNode(this.props.index + 1)}>
-										Move Down
-									</Button>
-								)}
+								<Button onClick={() => this.props.moveNode(this.props.index - 1)}>Move Up</Button>
+								<Button onClick={() => this.props.moveNode(this.props.index + 1)}>Move Down</Button>
```
Comment (qwertynerd97) [inline packages/app/obojobo-document-engine/src/scripts/oboeditor/components/navigation/more-info-box.js]: Use the index to hide buttons that would not have an effect

**Comment by vutoan1245** [comment]
New changes:

- Move Up and Move Down buttons are disabled when applicable
- You can no longer move an Assessment
- A node cannot move pass multiple-choice options in Question

**Comment by vutoan1245** [comment]
Resolved conflict. Not sure why Prettier formatted many unrelated files.

File: packages/app/obojobo-document-engine/src/scripts/oboeditor/components/node/editor-component.js [hunk 2]
```diff
@@ -106,10 +119,9 @@ class Node extends React.Component {
 				{selected ? (
 					<MoreInfoBox
 						className="content-node"
+						index={index}
+						type={this.props.node.type}
```
Comment (iturgeon) [inline packages/app/obojobo-document-engine/src/scripts/oboeditor/components/node/editor-component.js:123]: it looks like `this.props.node.type` should be `this.props.element.type`

File: packages/app/obojobo-document-engine/src/scripts/oboeditor/components/node/editor-component.js [hunk 3]
```diff
@@ -78,7 +78,20 @@ class Node extends React.Component {
...
 	}
 
+	moveNode(index) {
+		const editor = this.props.editor
+
+		editor.moveNodeByKey(this.props.node.key, this.props.parent.key, index)
+	}
+
 	render() {
+		let size = this.props.parent.nodes.size
```
Comment (iturgeon) [inline packages/app/obojobo-document-engine/src/scripts/oboeditor/components/node/editor-component.js:88]: it looks like `this.props.parent` doesn't exist anymore

**Comment by iturgeon** [comment]
Fixed up an ready again for testing.  Discovered a bug upstream in dev/10 #1266

**Comment by zachberry** [comment]
Also, want to check that this will be compliant with #1212

**Comment by vutoan1245** [comment]
It should be ready now. Not sure why it closes the second time but I think we can put into another issue.

**Pull request merged**